MdiChild/MdiChild.cpp

```diff
@@ -0,0 +1,66 @@
+//
+// Created by dawid on 12.02.18.
+//
+
+#include <QtGui/QTextDocument>
+#include <QtGui/QImageReader>
+#include "MdiChild.h"
+
+MdiChild::MdiChild()
+{
+  setAttribute(Qt::WA_DeleteOnClose);
+  isUntitled = true;
+}
+void MdiChild::newFile()
+{
+  static int sequenceNumber = 0;
+
+  isUntitled = true;
+  openedFile = tr("drawing%1.jpg").arg(sequenceNumber++);
+  setWindowTitle(openedFile + "[*]");
+}
+bool MdiChild::loadFile(const QString &fileName)
+{
+  QImageReader imgRead(fileName);
+  if (imgRead.canRead())
+  {
+    image = imgRead.read();
+    LoadImage(image);
+    qDebug("Image readed");
+    return true;
+  } else
+  {
+    qDebug(imgRead.errorString().toStdString().c_str());
```

Comment:
Wrong.
`qDebug() << imgRead.errorString();`

Reply:
When I try use this construction:
error: invalid use of incomplete type ‘class QDebug’
         qDebug() << imgRead.errorString();
                       ^

Reply:
That's because you need to add:
`#include <QDebug>`